### ISSUE TITLE
accessibility-screen

### ISSUE THREAD

**Issue by LukeSchrez** [RogueDragoon/Chatbot#15]
As a user I would like:

- [x] I would like to see an accessibility screen implemented into the chatbot app, with an icon in the top left of the view window.
- [x] This icon when pressed should take me to an accessibility screen where I should see one slider and one touch switch.
- [x] The slider should be able to manipulate brightness on my phone, it does not have to update brightness on the simulator.
- [x] The touch switch should be able to activate or deactivate dark mode on the device depending on the user's preferences.
- [x] When I click the back button from the accessibility screen, I should be returned back to the chatbot screen.
- [x] When I return to the chatbot screen, my previous conversations should be retained and I can continue where I left off.
